quail/processes/wps_climdexInput_csv.py

```diff
@@ -234,19 +235,27 @@ def _handler(self, request, response):
             log_level=loglevel,
             process_step="process",
         )
-        ci = climdex.climdexInput_csv(
-            **params,
-            base_range=robjects.r(base_range),
-            na_strings=na_strings,
-            cal=robjects.r(f"'{cal}'"),
-            n=n,
-            northern_hemisphere=northern_hemisphere,
-            quantiles=robjects.r(quantiles),
-            temp_qtiles=robjects.r(temp_qtiles),
-            prec_qtiles=robjects.r(prec_qtiles),
-            max_missing_days=robjects.r(max_missing_days),
-            min_base_data_fraction_present=min_base_data_fraction_present,
-        )
+
+        try:
+            ci = climdex.climdexInput_csv(
+                **params,
+                base_range=robjects.r(base_range),
+                na_strings=na_strings,
+                cal=robjects.r(f"'{cal}'"),
+                n=n,
+                northern_hemisphere=northern_hemisphere,
+                quantiles=robjects.r(quantiles),
+                temp_qtiles=robjects.r(temp_qtiles),
+                prec_qtiles=robjects.r(prec_qtiles),
+                max_missing_days=robjects.r(max_missing_days),
+                min_base_data_fraction_present=min_base_data_fraction_present,
+            )
+        except RRuntimeError as e:
+            err = ProcessError(msg=e)
```

Comment:
When you pass in `e` to `ProcessError` does the original error type get listed in the message?

Reply:
it's in the message, yes